jina/orchestrate/deployments/config/k8slib/kubernetes_deployment.py

```diff
@@ -85,14 +88,20 @@ def get_deployment_yamls(
         'shard_id': f'\"{shard_id}\"' if shard_id is not None else '\"\"',
         'pod_type': pod_type,
         'protocol': str(protocol).lower() if protocol is not None else '',
+        'volume_path': volumes[0] if volumes is not None else None
```

Comment:
Why do we only mount the first volume ? I guess volume mounts in k8s yaml supports several entries ?

Reply:
I did not know about this. Will change

Reply:
What's the use case of allowing users to mount several volumes? This api is already hacky because it doesn't support mounting volumes from user defined Secrets and ConfigMaps. I would rather let them create sub directories on the mounted volume rather than mixing the usage of Pod level volumes. 

If we allowed SSD instance types then maybe it makes sense to create volumes backed by the local SSD disk.

Reply:
after talking to @tarrantro, it seems not so trivial to do so, I will just document it

Reply:
@girishc13 if the use case doesn't make sense for k8s, then it doesn't make sense for jina right ?
but anyways, I think documenting this behavior is enough